src/WebJobs.Script.WebHost/Middleware/SystemTraceMiddleware.cs

```diff
@@ -46,13 +46,22 @@ public async Task Invoke(HttpContext context)
 
             await _next.Invoke(context);
 
+            LogIfOffline(request, scriptHostManager.State);
```

Comment:
The issue says you're trying to log this no "ping" requests - which ping requests are you talking about? E.g. there might not be any http traffic coming into the app if this is a dedicated function app, and there are no http functions (e.g. only queue trigger functions).

Reply:
@mathewc, I was talking about `/admin/host/ping` requests. Does that not get called by AlwaysOn for dedicated function app. Is there a different request for that?

Reply:
Yes, verified that the AlwaysOn ping for Function apps is made to admin/host/ping rather than root. I was unaware of that.